TryConvertInstaller.targets

```diff
@@ -9,7 +9,7 @@
       <!-- By default, NuGet will not include files starting with '.' or ending with '.nupkg'. Since we are including try-convert via dotnet tools, some of the files fit into this. -->
       <NoDefaultExcludes>true</NoDefaultExcludes>
       <ToolsDirectory>$(MSBuildThisFileDirectory).tools\</ToolsDirectory>
-      <TryConvertVersion>0.7.216201</TryConvertVersion>
+      <TryConvertVersion>0.7.217901</TryConvertVersion>
```

Comment:
Were there any command line arg changes? Or does it just build on the previous ones? I remember the change checked the state of an arg, but not sure if it's one we're already passing.

Reply:
Great question, I actually had to double check this because I was expecting the message to show up.

The change checked the state of the keepTFM arg and uses that to determine if a warning message should be displayed. In this case the warning is not displayed because we're not changing TFM.